lib/dataset/market1501.py

```diff
@@ -0,0 +1,113 @@
+from __future__ import division, print_function, absolute_import
+import re
+import glob
+import os.path as osp
+import warnings
+from lib.dataset import ImageDataset
+
+
+class Market1501(ImageDataset):
+    """Market1501.
+
+    Reference:
+        Zheng et al. Scalable Person Re-identification: A Benchmark. ICCV 2015.
+
+    URL: `<http://www.liangzheng.org/Project/project_reid.html>`_
+
+    Dataset statistics:
+        - identities: 1501 (+1 for background).
+        - images: 12936 (train) + 3368 (query) + 15913 (gallery).
+
+    以 0001_c1s1_000151_01.jpg 为例
+    1） 0001 表示每个人的标签编号，从0001到1501；
+    2） c1 表示第一个摄像头(camera1)，共有6个摄像头；
+    3） s1 表示第一个录像片段(sequece1)，每个摄像机都有数个录像段；
+    4） 000151 表示 c1s1 的第000151帧图片，视频帧率25fps；
+    5） 01 表示 c1s1_001051 这一帧上的第1个检测框，由于采用DPM检测器，对于每一帧上的行人可能会框出好几个bbox。00 表示手工标注框
+    """
+    _junk_pids = [0, -1]
+    dataset_dir = 'market1501'
+    dataset_url = 'http://188.138.127.15:81/Datasets/Market-1501-v15.09.15.zip'
```

Comment:
what is format of the dataset and its annotation ? Since you are not using Coco API or VoC api, do you inlcude logics to process it ? and test it with any PaddleDetection model ?

Reply:
No I have not. I will update new version following the discussion we have previously.